website/content/en/docs/tasks/deprov-nodes.md

```diff
@@ -1,36 +1,84 @@
+    Keep in mind that a small NodeExpiry results in a higher churn in cluster activity. So, for example, if a cluster brings up all nodes at once, all the pods on those nodes would fall into the same batching window on expiration.
+    {{% /alert %}}
+    
+* **Node deleted**: You could use `kubectl` to remove a single Karpenter node:
 
-## Delete Node
+    ```bash
+    kubectl delete <nodename>
+    ```
 
-Karpenter changes the behavior of `kubectl delete node`. Nodes will be drained, and then the underlying instance will be deleted.
+    Or delete all Karpenter nodes at once:
 
-## Disruption Budget
+    ```bash
+    kubectl delete -l <provisioner label>
+ 
+    ```
 
-Karpenter respects Pod Disruption Budgets. Review what [disruptions are](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/), and [how to configure them](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
+Whether through node expiry or manual deletion, Karpenter seeks to follow graceful termination procedures as described in Kubernetes [Graceful node shutdown](https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdow) documentation.
 
-Generally, pod workloads may be configured with `.spec.minAvailable` and/or `.spec.maxUnavailable`. Karpenter provisions nodes to accommodate these constraints. 
+{{% alert title="Note" color="primary" %}}
+By adding the finalizer, Karpenter improves the default Kubernetes process of node deletion.
+When you run `kubectl delete node` on a node without a finalizer, the node gets deleted in the API server but the instance keeps running.
+The kubelet isn’t watching for its own existence, so if a node is deleted the kubelet doesn’t terminate itself.
+All the pod objects get deleted a minute later, from a garbage collection process because the pods’ nodes are gone.
+{{% /alert %}}
 
-## Emptiness
+## What can cause deprovisioning to fail?
 
-Karpenter will delete nodes (and the instance) that are considered empty of pods. Daemonset pods are not included in this calculation. 
+There are a few cases where requesting to deprovision a Karpenter node will fail. These include Pod Disruption Budgets and pods that have the `do-not-evict` annotation set.
 
-## Expiry
+### Disruption budgets
 
-Nodes may be configured to expire. That is, a maximum lifetime in seconds starting with the node joining the cluster. Review the `ttlSecondsUntilExpired` field of the [provisioner API](../../provisioner/).
+Karpenter respects Pod Disruption Budgets (PDBs) when provisioning nodes and can prevent a node from being deprovisioned to maintain those disruption budgets.
+Kubernetes PDBs let you specify how much of a Deployment, ReplicationController, ReplicaSet, or StatefulSet must be protected from disruptions when pod eviction requests are made. 
+
+PDBs can be used to strike a balance by protecting the application's availability while still allowing a cluster operator to manage the cluster.
```

Comment:
This should follow in line with this, right? https://github.com/aws/karpenter/pull/1137 
```suggestion
PDBs can be used to strike a balance by protecting the application's availability while still allowing a cluster admin to manage the cluster.
```